### PR TITLE
Add missing cast(string) in _d_assert_fail

### PR DESCRIPTION
**Pull request by nordlow**
See https://forum.dlang.org/post/eijbfurdbbnhvepfaatb@forum.dlang.org

### CODE REVIEW

File: src/object.d [hunk 1]
```diff
@@ -4842,5 +4842,5 @@ string _d_assert_fail(string comp, A, B)(A a, B b) @nogc @safe nothrow pure
     n += token.length;
     buffer[n++] = ' ';
     buffer[n .. n + valB.length] = valB;
-    return buffer;
+    return cast(string)buffer;
```
Comment (MoonlightSentinel) [inline src/object.d]: Local trusted?

```suggestion
    return (() @trusted => cast(string) buffer)();
```

**Comment by kinke** [comment]
Please target stable, so that this makes it into 2.085.1.

**Comment by nordlow** [comment]
Sorry for asking but...how do I target stable `instead` of `master`?

File: src/object.d (hunk 1 above)
Comment (nordlow) [reply src/object.d]: Done.

**Comment by wilzbach** [comment]
Changed the target branch for you on GitHub. However, you still need to run sth. like this to update your branch:

```
git rebase --onto upstream/stable upstream/master
```

(then the other commits will disappear).
I'm currently travelling. Sorry.

**Comment by nordlow** [comment]
Done, @wilzbach :)

**Comment by wilzbach** [comment]
No tests??

**Pull request merged**